docs/uma-tokenholders/general-kpi.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Launching Generalized KPI Options
```

Comment:
Small nit: the KPI options themselves are not generalized, just the price identifier being used. Maybe change this to "Using the Generalized KPI Options Price Identifier"

Reply:
updated